terracumber_config/tf_files/tfvars/PR-TEST-manager43.tfvars

```diff
@@ -0,0 +1,14 @@
+############ Spacewalk unique variables ###########
+
+IMAGE                  = "sles15sp4o"
+IMAGES                 = ["rocky8o", "opensuse154o", "sles15sp4o", "ubuntu2204o"]
+PRODUCT_VERSION        = "4.3-nightly"
```

Comment:
Why in Uyuni we have a "uyuni-pr" but in SUMA we can reuse "4.3-nightly" ?

Reply:
Because we don't have a support for 4.3-pr yet or ever ( need to check with Jordi )